base/special/trig.jl

```diff
@@ -1294,5 +1302,5 @@ end
 
 Compute the inverse tangent of `y` or `y/x`, respectively, where the output is in degrees.
 """
-atand(y)    = rad2deg(atan(y))
-atand(y, x) = rad2deg(atan(y,x))
+atand(y)    = rad2deg.(atan(y))
```

Comment:
@ViralBShah Thank you for your comment. The PR title and news.md was missing `atand`, but it supports a square matrix by this change.